spi/common/core-spi/src/main/java/org/eclipse/edc/spi/entity/StateEntityManager.java

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.spi.entity;
+
+/**
+ * Is a long-running process that takes care of managing entity and their status.
```

Comment:
Another nit to comply with Javadoc recommendations: "A long-running process that manages entities and their states"